### ISSUE TITLE
Questions about the degradation process for BDx4 low-resolution input

### ISSUE THREAD

**Issue by yhjoker** [open-mmlab/mmediting#551]
For BIx4, we use MATLAB to resize the images: `lq = imresize(gt, 0.25)`.

For BDx4, you can find the MATLAB script [here](https://github.com/ckkelvinchan/BasicVSR-IconVSR/blob/main/BD_degradation.m).

_Originally posted by @ckkelvinchan in https://github.com/open-mmlab/mmediting/issues/487#issuecomment-899047094_

**Comment by yhjoker** [comment]
Hello there, I am currently trying to train the basicvsr model on Vimeo-90K( BD degradation ) and find that there are slightly differences between the downsampling functions used in DUF and those used in Basicvsr ( for example, gussian smooth kernel size ). Any reason why we choose to use the Matlab version in basicvsr training?

**Comment by ckkelvinchan** [comment]
The `sigma` we used is the same. About the kernel size, I followed previous works (I am not sure whether I copied from RCAN but RCAN uses this setting [here](https://github.com/yulunzhang/RCAN/blob/master/RCAN_TestCode/Prepare_TestData_HR_LR.m#L84)).

**Comment by yhjoker** [comment]
> The `sigma` we used is the same. About the kernel size, I followed previous works (I am not sure whether I copied from RCAN but RCAN uses this setting [here](https://github.com/yulunzhang/RCAN/blob/master/RCAN_TestCode/Prepare_TestData_HR_LR.m#L84)).

Thanks for the answer！ @ckkelvinchan I am trying to retrain the basicvsr model using the degradation function provided by EDVR ( here is the [link ](https://github.com/xinntao/EDVR/blob/b02e63a0fc5854cad7b11a87f74601612e356eff/basicsr/data/data_util.py#L299)) but find the training process can not convergence. I use the pretrained network-sintel-final.pytorch model for SPynet at [here](https://github.com/sniklaus/pytorch-spynet/blob/master/run.py#L92) and basicvsr_vimeo90k_bd.py config file( only [spynet_pretrained path](https://github.com/open-mmlab/mmediting/blob/b68821930da58c63e6afb540581b3041a654a6a2/configs/restorers/basicvsr/basicvsr_vimeo90k_bd.py#L10) is modified ), is there any tools to help locate the training problems in mmediting repo?

**Comment by ckkelvinchan** [comment]
Could you provide me with more information? For example, the loss curve and how you generate the data.

**Comment by yhjoker** [comment]
> Could you provide me with more information? For example, the loss curve and how you generate the data.

After adjusting the learning rate to 1e-4, things start to get better. I use the degradation function proposed by DUF to generate LR input( the duf_downsample provided in [EDVR repo](https://github.com/xinntao/EDVR/blob/b02e63a0fc5854cad7b11a87f74601612e356eff/basicsr/data/data_util.py#L299) ). Other parameters might influence the training process are:
- learning rate : 1e-4
- samples_per_gpu : 2
- gpus : 4
- total_iters : 300000

It seems loss is decreasing and validation psnr is increasing( very unstable, though ). The loss curve and validation results for 250000/300000 training process is showed as following. Maybe a larger total_iter setting is required to get a promising result. Thanks again for your kind reply!
![Screenshot from 2021-09-28 20-11-53](https://user-images.githubusercontent.com/36630680/135084790-9ce43191-c8a7-4e5c-99c9-49b640c55b33.png)
![Screenshot from 2021-09-28 20-12-08](https://user-images.githubusercontent.com/36630680/135084824-6012ce7c-c5f0-46ac-af1f-c3268d4a7a44.png)

**Comment by ckkelvinchan** [comment]
I think a PSNR of about 25 is a bit low. Did you compare this result to the output upsampled by bicubic interpolation?
